app.rb

```diff
@@ -0,0 +1,172 @@
+require './book'
+require './teacher'
+require './student'
+require './rental'
+
+class App
+  attr_reader :books, :people, :rental
+
+  def initialize
+    @books = []
+    @people = []
+    @rental = []
+  end
+
+  def run
+    show_menu
+
+    menu_list = {
+      '1' => method(:book_list),
+      '2' => method(:people_list),
```

Comment:
- [ ] It will be great to return the `Name`, `age` and `id` when listing all people. At the moment, the only thing shown is the name. kindy fix this implementation.

Reply:
Thank You for your comments, but all of them are not specified in assigned tasks or requirements for this project, specially the first one. Which te task specifically says create a teacher or student, not a person.